lib/model.js

```diff
@@ -1466,10 +1466,8 @@ Model.prototype.findById = function(param, options) {
     throw new Error('Argument passed to findById is invalid: '+param);
   }
 
-  // Bypass a possible overloaded findAll.
-  return Model.prototype.findAll.call(this, _.defaults(options, {
-    plain: true
-  }));
+  // Bypass a possible overloaded findOne
+  return Model.prototype.findOne.call(this, options);
```

Comment:
This change wouldn't actually have done anything as far as i can tell? findAll does not add a limit, neither does findById.